packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import type { NextPage } from "next";
+import { useTheme } from "next-themes";
+import Batch4AddressDisplay from "~~/app/builders/_components/Batch4AddressDisplay";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+
+const BuildersPage: NextPage = () => {
+  const { data: checkedInEvents, isLoading } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 118484927n,
+    watch: true,
```

Comment:
Also, I think we can remove this. It's going to cause a lot of RPC calls... and we don't need to be that dynamic in this case :D